src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -228,9 +233,10 @@ class SaveInProgressIntro extends React.Component {
             <div className="usa-alert-text">
               {this.props.displayNonVeteranMessaging ? (
                 <p>
-                  By signing in, you can save your work in progress.
-                  You&rsquo;ll have {retentionPeriod} from when you start or
-                  make updates to your {appType} to come back and finish it.
+                  By signing in, you can save your work in progress.{' '}
+                  You&rsquo;ll have {retentionPeriod} from{' '}
+                  {retentionPeriodStart} your {appType} to come back and finish
+                  it.
```

Comment:
The acceptance criteria copy is:
`You'll have one year from when you start or make updates to your application to come back and finish it.` 
but this update copy is:
`You’ll have a year from when you start your application to come back and finish it.`

Reply:
fixed.  i checked the copy after starting the pr and didn't go back to update the actual code 🤦